app/models/course/condition/level.rb

```diff
@@ -1,4 +1,8 @@
 class Course::Condition::Level < ActiveRecord::Base
   acts_as_condition
   validates :minimum_level, numericality: { greater_than: 0 }
+
+  def title
+    I18n.t('course.condition.level.title', value: minimum_level)
```

Comment:
It's not an error message or a property of ActiveRecord. Wouldn't it belong more in the condition namespace?